src/Makefile.am

```diff
@@ -317,6 +317,10 @@ install-data-hook:
 	        rm "$(DESTDIR)$(dictdir)/$$file" || exit $$?; \
 	    fi; \
 	done
+	$(MKDIR_P) "$(DESTDIR)$(SYSTEMD_USER_UNIT_DIR)/gnome-session.target.wants/"
+	$(LN_S) -f "$(DESTDIR)$(SYSTEMD_USER_UNIT_DIR)/org.freedesktop.IBus.service" \
+		"$(DESTDIR)$(SYSTEMD_USER_UNIT_DIR)/gnome-session.target.wants/"
```

Comment:
I think the relative symlink is better than the absolute symlink.

```
( \
  cd "$(DESTDIR)$(SYSTEMD_USER_UNIT_DIR)/gnome-session.target.wants" ; \
  $(LN_S) -f "../org.freedesktop.IBus.session.GNOME.service" .; \
)
```

Reply:
Oops, indeed. Done.